java/adapter/orc/pom.xml

```diff
@@ -160,6 +160,39 @@ under the License.
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration combine.self="override">
```

Comment:
can we possibly simplify this with `combine.children="append"`?

Reply:
Right, let me check.

Reply:
I updated, lets check the CIs (locally passing)